packages/stories-block/src/block/block-types/single-story/embedControls.js

```diff
@@ -151,7 +151,7 @@ const EmbedControls = (props) => {
                       )}
                 </p>
                 {hasPoster && (
-                  <Button onClick={onRemovePoster} isTertiary>
+                  <Button className="remove" onClick={onRemovePoster}>
```

Comment:
Nit: let's use `web-stories-embed-poster-remove` as the class name, in line with the `web-stories-embed-poster-label` class name above.

Reply:
Sounds good -- pushed a commit.